### PR TITLE
Fat 863/c347623

### PR DESCRIPTION
**Pull request by kimkate303**
https://foliotest.testrail.io/index.php?/cases/view/347623

### CODE REVIEW

**Review by TetianaParanich — APPROVED** [review]

File: cypress/integration/recieve-notice/recieve-notice-checkin.spec.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      });
+
+    CirculationRules.deleteRuleApi(testData.baseRules);
+    NoticePolicyApi.deleteApi(testData.ruleProps.n);
+    NoticePolicyTemplateApi.getViaApi({ query: `name=${noticePolicyTemplate.name}` }).then((templateId) => {
+      NoticePolicyTemplateApi.deleteViaApi(templateId);
+    });
+  });
+
+  it('C347623 Check that user can receive notice with multiple items after finishing the session "Check in" by clicking the End Session button', { tags: [testTypes.smoke, devTeams.vega] }, () => {
```
Comment (dmitriyfolio) [inline cypress/integration/recieve-notice/recieve-notice-checkin.spec.js:142]: check testgroup of this TC in TestRail

File: cypress/integration/recieve-notice/recieve-notice-checkin.spec.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    NewNoticePolicyTemplate.startAdding();
+    NewNoticePolicyTemplate.checInitialState();
+    NewNoticePolicyTemplate.create(noticePolicyTemplate);
+    NewNoticePolicyTemplate.addToken(noticePolicyTemplate);
+    noticePolicyTemplate.body += '{{item.title}}';
+    NewNoticePolicyTemplate.save();
+    NewNoticePolicyTemplate.checkAfterSaving(noticePolicyTemplate);
```
Comment (dmitriyfolio) [inline cypress/integration/recieve-notice/recieve-notice-checkin.spec.js:149]: let's expand object properties into methods parameters to clear scope of testing. Or add check that that object contains expected properties...

File: cypress/integration/recieve-notice/recieve-notice-checkin.spec.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    NewNoticePolicyTemplate.checkTemplateActions(noticePolicyTemplate);
+
+    cy.visit(settingsMenu.circulationPatronNoticePoliciesPath);
+    NewNoticePolicy.waitLoading();
+    NewNoticePolicy.startAdding();
+    NewNoticePolicy.checInitialState();
+    NewNoticePolicy.create(noticePolicy);
+    NewNoticePolicy.addNotice(noticePolicy);
```
Comment (dmitriyfolio) [inline cypress/integration/recieve-notice/recieve-notice-checkin.spec.js:157]: can't see checking of ""Always sent at the end of a session and loans are bundled into a single notice for each patron." message appeared in the Notice 1"

**Review by dmitriyfolio — APPROVED** [review]

File: cypress/integration/recieve-notice/recieve-notice-checkin.spec.js (hunk 1 above)
Comment (kimkate303) [reply cypress/integration/recieve-notice/recieve-notice-checkin.spec.js:142]: yes it is vega

**Pull request merged**